deployment/docker-compose.yml

```diff
@@ -1,17 +1,38 @@
 version: '3.7'
 services:
+  backend:
+    image: "ghcr.io/serokell/edna-backend:latest"
+    depends_on:
+      - postgres
+    volumes:
+      - ./config.yaml:/config.yaml
+    networks:
+      - edna-network
+    ports:
+      - 9000:9000
+    command: --config /config.yaml
+  frontend:
+    image: "ghcr.io/serokell/edna-frontend:latest"
+    depends_on:
+      - backend
+    ports:
+      - 8080:80
+    networks:
+      - edna-network
   postgres:
-    image: "postgres"
+    image: postgres
     container_name: postgres
     ports:
-      - "5432:5432"
+      - 5432:5432
     environment:
       POSTGRES_USER: ednauser
       POSTGRES_PASSWORD: ednapass
       POSTGRES_DB: ednadb
     volumes:
-      - ./postgres:/var/lib/postgresql/data/
+      - edna-db-data:/var/lib/postgresql/data/
```

Comment:
While a named volume is "cleaner", I suppose, it also means you don't have very easy manual access to the data. You have to run, say, a busybox container and mount the volume to manipulate it. The reason I used a host path is so that you could easily reset or manipulate the database during development.

Reply:
I treat `postgres` folder as black box that I shouldn't touch and use `psql` to interact with DB. AFAIK @pva701 and @sashasashasasha151 do something similar (they use IDE plugin instead of plain `psql`, but it shouldn't matter). So if I want to reset DB, I can do `\i sql/drop.sql;` or run `edna-server` with `init-with-drop`. So I think it's not a problem.

Reply:
I'm not sure if named volumes are cleaner, I just noticed the problems I described in the commit message and wanted to help with it while you were not around, but if you think there's a better way, feel free to change it back, or change it to something else.

Dropping a named volume is simple with `docker-compose down -v` or `docker volume rm`, but inspecting or changing it is harder, yes.